### PR TITLE
Index, installation and getting started

### PR DESCRIPTION
**Pull request by marco-2023**
This PR has the three abovementioned pages.

### CODE REVIEW

File: docs/notebooks/getting_started.ipynb [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Printing atom properties using the script:**\n",
+    "\n",
+    "The data is printed as human readable following the JSON format. This can be latter parsed and used \n",
+    "in other scripts or programs. The following cell shows how to print the data for the Carbon atom with\n",
+    "charge 0 and spin multiplicity 3 (ground state). Due to the large amount of data, the output stored\n",
+    "in the variable c_0_3_json_output sn only three properties are parsed and printed."
```
Comment (gabrielasd) [inline docs/notebooks/getting_started.ipynb:80]: This last sentence looks strange to me.
Could it be:
Due to the large amount of data, from the output stored in the variable c_0_3_json_output only three properties are parsed and printed.

File: docs/notebooks/getting_started.ipynb [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+    "done"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use as a Python library\n",
+    "\n",
+    "The real power of AtomDB is showcased when it is used as a library. The library provides a simple API to access atomic properties and even some pro-molecular properties this is done through the `Species` object, which is used to access atomic properties. \n",
```
Comment (gabrielasd) [inline docs/notebooks/getting_started.ipynb:118]: Another minor issue:
``...some pro-molecular properties. This is done..."

**Review by gabrielasd — APPROVED** [review]
@marco-2023 this looks really good. Thanks!

File: docs/notebooks/getting_started.ipynb (hunk 1 above)
Comment (marco-2023) [reply docs/notebooks/getting_started.ipynb]: Done! Thanks!!

File: docs/notebooks/getting_started.ipynb (hunk 2 above)
Comment (marco-2023) [reply docs/notebooks/getting_started.ipynb]: Done! Thanks!!

**Pull request merged**